Makefile

```diff
@@ -13,8 +13,11 @@ grunt:
 grunt-process-lists:
 	grunt execute:preProcessLists --browser=$(browser) --type=$(type)
 
+build-test-files:
+	grunt browserify:backgroundTest --browser=$(browser) --type=$(type)
+
 grunt-dev:
-	cp -r test build/$(browser)/dev/
+	cp -Lr test build/$(browser)/dev/
```

Comment:
The `-L` flag resolves the symlinks (for `img` and `data`) and creates copies of the files. For some reason I was seeing the tests fail without this. I didn't look into it further.